components/automate-ui/src/app/modules/infra-proxy/create-org-modal/create-org-modal.component.ts

```diff
@@ -68,6 +68,6 @@ export class CreateOrgModalComponent implements OnInit, OnChanges {
   }
 
   private isNavigationKey(event: KeyboardEvent): boolean {
-    return event.key === 'Shift' || event.key === 'Tab';
+    return event.key === 'Shift' || event.key === 'Tab' || event.key === 'Enter';
```

Comment:
In some of the modals, this may need to be adjusted so that when using keyboard navigation and the "Enter" key to open the modal, enter isn't recorded.  Nothing happens in the view, but an error is thrown in the console.

Reply:
Good catch. I would like to suggest that this `isNavigationKey` should go into a utilities folder (perhaps under automate-ui/src/app/helpers/) rather than redefine it 10 places. Want an easy way to see the variations? Use semgrep!
```
$ semgrep -e 'isNavigationKey(...) {...}' --lang ts
src/app/components/create-object-modal/create-object-modal.component.ts
139:  private isNavigationKey(event: KeyboardEvent): boolean {
140:    return event.key === 'Shift' || event.key === 'Tab';
141:  }

src/app/modules/infra-proxy/create-chef-server-modal/create-chef-server-modal.component.ts
53:  private isNavigationKey(event: KeyboardEvent): boolean {
54:    return event.key === 'Shift' || event.key === 'Tab';
55:  }

src/app/modules/infra-proxy/create-org-modal/create-org-modal.component.ts
70:  private isNavigationKey(event: KeyboardEvent): boolean {
71:    return event.key === 'Shift' || event.key === 'Tab' || event.key === 'Enter';
72:  }

src/app/page-components/create-user-modal/create-user-modal.component.ts
154:  private isNavigationKey(event: KeyboardEvent): boolean {
155:    return event.key === 'Shift' || event.key === 'Tab';
156:  }

src/app/pages/+compliance/+node-credentials/create-node-credential-modal/create-node-credential-modal.component.ts
150:  private isNavigationKey(event: KeyboardEvent): boolean {
151:    return event.key === 'Shift' || event.key === 'Tab';
152:  }

. . .
```

Reply:
this is beautiful ✨ 

And great idea, I'll incorporate that in the final product